hedera-node/src/main/java/com/hedera/services/ledger/interceptors/AccountsCommitInterceptor.java

```diff
@@ -70,6 +114,10 @@ private void trackBalanceChangeIfAny(
 	) {
 		if (accountChanges.containsKey(AccountProperty.BALANCE)) {
 			final long newBalance = (long) accountChanges.get(AccountProperty.BALANCE);
+			if (merkleAccount != null && (accountNum == STAKING_FUNDING_ACCOUNT_NUMBER)) {
```

Comment:
The staking reward account must always exist, so `null`-check is unnecessary.

Reply:
Fixed !